manifests/base.pp

```diff
@@ -1,13 +1,5 @@
-# Class: rsyslog::base
-#
-# Description
-# -----------
-#
-# This class manages the base installation for rsyslog
+# @summary This class manages the base installation for rsyslog
```

Comment:
Is this also a private class or is it public?

Reply:
Should probably be private since it's not directly called by another class